### ISSUE TITLE
Can you provide vilbert finetuned checkpoint of MAVEx stage1?

### ISSUE THREAD

**Issue by ParadoxZW** [jialinwu17/MAVEX#9]
Thanks for your valuable work!
I'm conducting some experiments on OKVQA. It will be helpful if you can provide the finetuned vilbert model of MAVEx stage1.

**Comment by jialinwu17** [comment]
Hi, sorry for the late response and your interests in our work. Due to a recent install of storage limit for our school's google drive, the old way of sharing features doesn't work. I am working on finding new ways to share features.

**Comment by ParadoxZW** [comment]
Or can you share the files of candidates with their confidence on train and test set, through other way, for example, BaiduYun?

**Comment by yangxingrui** [comment]
@ParadoxZW Hello, have you obtained the retrieved features through other methods?
